db/seeds.rb

```diff
@@ -41,12 +41,12 @@
 ])
 
 Answer.create([
-  {id: 1, answer: 'Yahtzee!'},
-  {id: 2, answer: 'Painting!'},
-  {id: 3, answer: 'Gaming!'},
-  {id: 4, answer: 'Write code!'},
-  {id: 5, answer: 'Biking!'},
-  {id: 6, answer: 'Knitting!'},
+  {id: 1, answer: 'Yahtzee!', question_id: 1},
```

Comment:
There should be a way to have these inside of the Question.create() function. 

It might be something like:

Question.create([
  {id: 1, question: 'Play a board game or go biking?', order: 3, answers: ["answer1"]}
]) 

or something